synapse/axon.py

```diff
@@ -995,7 +1031,10 @@ async def iterMpkFile(self, sha256):
     async def readlines(self, sha256):
         remain = ''
         async for byts in self.get(s_common.uhex(sha256)):
-            text = remain + byts.decode()
+            try:
```

Comment:
we should make this use the open() / executor / queue mechanism too while we're here